extensions/ql-vscode/.eslintrc.js

```diff
@@ -5,16 +5,23 @@ module.exports = {
     sourceType: "module",
     project: ["tsconfig.json", "./src/**/tsconfig.json", "./gulpfile.ts/tsconfig.json", "./scripts/tsconfig.json", "./.storybook/tsconfig.json"],
   },
-  plugins: ["@typescript-eslint"],
+  plugins: [
+    "github",
+    "@typescript-eslint"
+  ],
   env: {
     node: true,
     es6: true,
   },
   extends: [
     "eslint:recommended",
-    "plugin:@typescript-eslint/recommended",
+    "plugin:github/internal",
```

Comment:
I don't think we need the internal config, those rules seem to be really specific to the internal applications: https://github.com/github/eslint-plugin-github/blob/ccbf6ce0c989a1acc782dd978f566fb954d7f513/lib/configs/internal.js